src/sentry/static/sentry/app/components/forms/selectControl.jsx

```diff
@@ -113,6 +113,10 @@ const StyledSelect = styled(React.forwardRef(forwardRef))`
     }
   }
 
+  &.Select--single.is-disabled .Select-control .Select-value .Select-value-label {
+    color: ${p => p.theme.gray1};
```

Comment:
Do we have a semantic theme item for disabled?

Reply:
We do! I can update the other form controls to use that colour.

Reply:
Done now.